azkaban-common/src/main/java/azkaban/executor/ExecutionControllerUtils.java

```diff
@@ -46,7 +46,10 @@
   private static final Logger logger = LoggerFactory.getLogger(
       ExecutionControllerUtils.class);
   private static final String SPARK_JOB_TYPE = "spark";
-  private static final String APPLICATION_ID = "${application.id}";
+  static final String APPLICATION_ID_OLD = "${application.id}";
```

Comment:
Can you please rename it to OLD_APPLICATION_ID and NEW_APPLICATION_ID?

Reply:
Will do